File: leetcode/1easy/권민서/1784.java
class Solution {
    public boolean checkOnesSegment(String s) {
        boolean foundZero = false;

        for (char c : s.toCharArray()) {
            if (c == '0') {
                foundZero = true;
            } else if (foundZero) {
                return false;
            }
        }

        return true;
    }
}

File: leetcode/1easy/임강민/1784.py
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution:
=======
class Leetcode.이재훈.Solution:
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    def checkOnesSegment(self, s: str) -> bool:
        cnt = 0
        for i in range(len(s) - 1):
            if s[i] != s[i + 1]:
                cnt += 1
        if cnt >= 2:
            return False
        else:
            return True

